src/config/voice.c

```diff
@@ -47,32 +47,24 @@ static int ini_handler(void* user, const char* section, const char* name, const
         duration = atoi(ptr + 1);
         }
 
-#if HAS_MUSIC_CONFIG
     if ( k && (req_id != MAX_VOICEMAP_ENTRIES) && (req_id == id) ) {
-        strlcpy(tempstring, value, k+1);
-        tempstring[MAX_VOICE_LABEL] = '\0';  // limit label length
+        current_voice_mapping.duration = duration;
+        if (HAS_MUSIC_CONFIG)
+            strlcpy(tempstring, value, k+1);  // return a requested mp3 label passed at *user to tempstring
```

Comment:
Can we use tempstr_cpy () here?

Reply:
no, we only copy the part of value which contains the label.